aws/src/database/multi_table/amend_tenure.py

```diff
@@ -11,17 +11,18 @@
 # Config - change these
 STAGE = Stage.HOUSING_PRODUCTION
 STAGE_PARAM = "production"  # Stage parameter for SSM
-ES_EXTENSION = "tenures"
-_update_date = datetime(day=31, month=1, year=2022).isoformat()
+_update_date = datetime(day=3, month=7, year=2023).isoformat()
```

Comment:
On line 19, if we take Tenure Id from user input, pehaps, we can also take the date time as well?
Seems a bit odd that one is hardcoded, while the other is not :thinking:

Reply:
Yeah, you may be right. I was thinking about this but I'm not sure how to make the CLI input for the date object be user-friendly. Comma separated or something?

Reply:
I agree with this point as well

Reply:
doesn't matter so long the prompt asking it tells what format to expect, ex:
> "Please enter new start date (format is mm/dd/yyyy): "

Reply:
sort of like AWS CLI configure command

Reply:
The current solution includes a text parser, so most formats should work fine. DD/MM/YYYY or DD-MM-YYYY or YYYY-MM-DD, etc. I tested it, it's very flexible. It's set to expect days before months in the day-month-year formats.

Reply:
There's also a confirm step right after it parses and converts the date.

Reply:
Added a hint to the input just for clarity:
```python
UPDATE_DATE = parser.parse(timestr=input("Enter a date (e.g. DD/MM/YYYY or YYYY-MM-DD): ").strip(),
                           parserinfo=parser.parserinfo(dayfirst=True)).isoformat()
```